src/herbivoresAndCarnivores.js

```diff
@@ -1,17 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.health = 100;
+    this.name = name;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = true;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  bite(animal) {
+    if (animal.hidden === false) {
```

Comment:
hidden value is already boolean 

```suggestion
    if (!animal.hidden) {
```

Reply:
I wrote it because I thought it was easier to read 😔 and with the exclamation mark, the tests do not pass